test/unit_convex_hull.jl

```diff
@@ -33,6 +33,20 @@ for N in [Float64, Rational{Int}]
     convex_hull!(points_2D) # check in-place version
     @test points_2D == [[N(-1), N(-1)], [N(1), N(0)], [N(1), N(1)], [N(0), N(1)]]
 
+    # three vertex case in 2 dimension
+    ccw_points = [[N(1), N(1)], [N(-1), N(1)], [N(-1), N(0)]]
+    ccw_p = convex_hull!(ccw_points)
+    ccw_expr = [[N(1), N(1)], [N(-1), N(1)], [N(-1), N(0)]]
+    @test  ccw_p == ccw_expr || ccw_p == circshift(ccw_expr, 1) || ccw_p == circshift(ccw_expr, 2) # points sorted in a counter-clockwise fashion
+    
+    cw_points = [[N(-1), N(1)], [N(1), N(1)], [N(-1), N(0)]]
+    cw_p = convex_hull!(cw_points)
+    cw_expr = [[N(-1), N(1)], [N(1), N(1)], [N(-1), N(0)]]
+    @test  cw_p == cw_expr || cw_p == circshift(cw_expr, 1) || cw_p == circshift(cw_expr, 2) # points sorted in clockwise fashion
+
+    aligned_points = [[N(1), N(1)], [N(2), N(2)], [N(3), N(3)]]
+    @test convex_hull!(aligned_points) == ispermutation([[N(1), N(1)], [N(2), N(2)]]) # points aligned
+
```

Comment:
Please add unit tests for the problems mentioned above (1-dimensional case, 3-dimensional case, three points on a vertical line).